proto/ibc/applications/interchain_accounts/v1/types.proto

```diff
@@ -25,3 +20,9 @@ message InterchainAccountPacketData {
   bytes data = 2;
```

Comment:
should we rename to something else? Data is a little vague?

Reply:
Hmm! Maybe `tx_bytes`, `tx_payload`... I'm not sure 🤔